extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -92,17 +94,17 @@ private function getInsightModule( $type, $config ) {
 	private function addLastRevision( $insightsPages, $displayPageviews ) {
 		global $wgLang;
 
-		foreach ( $insightsPages['pages'] as $key => $insight ) {
-			$insightsPages['pages'][$key]['metadataDetails'] = $this->getArticleMetadataDetails( $insight['metadata'] );
-			$insightsPages['pages'][$key]['editlink'] = $this->getEditUrl( $insight['link']['articleurl'] );
-			$insightsPages['pages'][$key]['edittext'] = $this->getArticleContributeText(
-				$insight['metadata'],
-				$insightsPages['type'] );
+		foreach ( $insightsPages[ 'pages' ] as $key => $insight ) {
+			$insightsPages[ 'pages' ][ $key ][ 'metadataDetails' ] = $this->getArticleMetadataDetails( $insight[ 'metadata' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "metadata" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)